pkg/kubelet/container/runtime.go

```diff
@@ -95,7 +95,7 @@ type Runtime interface {
 	// it is useful when doing SIGKILL for hard eviction scenarios, or max grace period during soft eviction scenarios.
 	KillPod(pod *v1.Pod, runningPod Pod, gracePeriodOverride *int64) error
 	// GetPodStatus retrieves the status of the pod, including the
-	// information of all containers in the pod that are visble in Runtime.
+	// information of all containers in the pod that are visible in Runtime.
```

Comment:
We have a spell check script - https://github.com/kubernetes/kubernetes/blob/master/hack/verify-spelling.sh.

Wondering why this wasn't caught by it...any ideas, @dixudx?